src/main/java/frc/robot/subsystems/Tramp/TrampSubsystem.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.Tramp;
+
+import org.littletonrobotics.junction.Logger;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class TrampSubsystem extends SubsystemBase {
+  
+    private final TrampSubsystemIO io; 
```

Comment:
@anvisingh12 Don't hard-code the subsystem name wherever you use it. You may need to refer to it many times, and the base class methods may need it. The base class has a getName() method. Override it here to return the name, then use getName() instead of the hard-coded name.